documentation/docs/steps/neoDeploy.md

```diff
@@ -54,3 +54,54 @@ steps:
       account: <myDeployAccount>
       host: hana.example.org
 ```
+
+## Example for invalidating the cache
+
+* Setting parameter `invalidateCache` to `true`, will clean up the cache of the deployed application to portal site and reflects the new changes. This is **applicable** only when the application is deployed and accessed via Portal service (Fiori Launchpad site).
+
+Setting this parameter to true, needs additional configurations to go through. Firstly, create an oauth credential as shown below:
+
+* **Oauth credential creation**:
+
+  * Select the oauth settings from your subaccount, create a new client with subscription to portal/nwc as shown in the following images:
+
+    ![Oauth client creation](../images/oauthClientCreation.png)
+
+  * Select the client tab, which provides an option to register a new client as shown below:
+
+    ![Register new client](../images/registerNewOauthClient.png)
+
+  * Click on the "Register New Client" button. After clicking, in the subscription field, select the portal landscape you would like to subscribe to, ex:  `portal/nwc` or `portal/sandbox` as shown below:
+
+    ![Portal subscription](../images/portalSubscription.png)
+
+  * In the "Authorization Grant" field, select "Client Credentials" from the drop down menu. Then, enter the "Secret" field with a user defined password and finally save the changes.
+
+After saving these changes, create an UsernamePassword type credential with client Id (obtained from registering new client) as username and client secret (Password enterred in the Secret field above) as password in Jenkins.
+
+* **Set Site Id**:
+
+After log in into the portal service from SAP CP, the user has a possibility to create a site directory, within which the deployed application resides. It is created in the following way:
+
+![Site directory](../images/siteDirectory.png)
```

Comment:
We do not need to explain how to create a site. If we go down that path we will have to explain too many details. The assumption, as I see it, is that the user already has a Launchpad site with at least one application deployed. We just need to guide them here in case they have not set a default and we need the siteId.